src/loop.jl

```diff
@@ -235,6 +263,14 @@ function interval_loop(factory_like, model_kwargs::Dict,
         if storage.enabled
             storage_e0 = results.storage_e[:,end]
         end
+        if demand_flexibility.enabled
+            if demand_flexibility.interval_balance || demand_flexibility.rolling_balance
+                init_shifted_demand = sum(
+                    results.load_shift_up - results.load_shift_dn,
+                    dims=2
+                )[:, 1]
```

Comment:
We are taking the first column from the summation here, what's in the rest columns? I presume it should be one column in total, no?

Reply:
I think the `dropdims` function may be a clearer way to do this.

Reply:
Thanks for letting me know! I'll check it out

Reply:
Done.